### PR TITLE
Donato's HACEP server build changes

### PR DESCRIPTION
**Pull request by mramendi**

### CODE REVIEW

File: doc-content/enterprise-only/openshift/hacep-client-proc.adoc [hunk 1]
```diff
@@ -22,17 +22,7 @@ oc extract secret/my-cluster-cluster-ca-cert --keys=ca.crt --to=- > src/main/res
...
-----
-ssl.keystore.location=keystore.jks
-ssl.truststore.location=keystore.jks
-ssl.keystore.password=password
-ssl.truststore.password=password
-bootstrap.servers=http://<serveraddress>
-----
-+
-Replace `<serveraddress>` with the address that the route for the Kafka server uses.
+. In the `src/main/resources` subdirectory of the project, edit the `configuration.properties` file. Replace `<bootstrap-hostname>`  with the address that the route for the Kafka server uses.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/openshift/hacep-client-proc.adoc]: This is two sentences so two steps. Consider something like this:
..of the project, open  the `configuration.properties` file and replace...

Also, is route a thing that uses an address? If not. try to rephrase 'that the route... uses...

File: doc-content/enterprise-only/openshift/hacep-server-proc.adoc [hunk 2]
```diff
@@ -17,10 +21,15 @@ You must prepare the source, build it, and then deploy it on {OPENSHIFT}.
 mvn clean install -DskipTests
 ----
 +
-. Use the OpenShift operator infrastructure to install Red Hat AMQ Streams. For information about installing Red Hat AMQ Streams, see https://access.redhat.com/documentation/en-us/red_hat_amq/7.3/html/using_amq_streams_on_openshift_container_platform/index[_Using AMQ Streams on OpenShift Container Platform_].
-. Using the `KafkaTopic` resource on {OPENSHIFT}, create the topics from all the YAML files in the `kafka-topics` subdirectory. 
+. Enable the OpenShift operator for Red Hat AMQ Streams and then create an AMQ Streams (kafka) cluster in the project. For information about installing Red Hat AMQ Streams, see https://access.redhat.com/documentation/en-us/red_hat_amq/7.5/html/using_amq_streams_on_openshift/[_Using AMQ Streams on OpenShift_].
+. To create the kafka topics that are required for operation of the server, remain in the `openshift-drools-hacep-distribution/sources` directory and run the following commands:
```
Comment (emmurphy1) [inline doc-content/enterprise-only/openshift/hacep-server-proc.adoc]: for the operation of the server
or 
for server operation

File: doc-content/enterprise-only/openshift/hacep-server-proc.adoc [hunk 3]
```diff
@@ -32,14 +41,27 @@ oc create -f kubernetes/role-binding.yaml
...
 +
 ----
-docker login --username=<user username>
-docker build -t <user_username>/openshift-kie-springboot:<tag> .  
-docker push <user_username>/openshift-kie-springboot:<tag>
+oc get is/openshift-kie-springboot -o template --template='{{range .status.tags}}{{range .items}}{{.dockerImageReference}}{{end}}{{end}}'
 ----
 +
-Alternatively, to use Podman for the build process, replace `docker` with `podman` in the commands. 
+. Edit the `kubernetes/deployment.yaml` file. Replace the existing image URL with the result of the previous command, removing all characters at the end starting with the `@` symbol, then add `:latest` to the line. For example:
```
Comment (emmurphy1) [inline doc-content/enterprise-only/openshift/hacep-server-proc.adoc]: Similar to before. Try to make one sentence or make two steps. In this case, consider:
Open the `kubernetes/deployment.yaml` file in a text editor.
Replace...

Comment (emmurphy1) [reply doc-content/enterprise-only/openshift/hacep-server-proc.adoc]: Well, you know my opinion on For example: Should be a sentence.

File: doc-content/enterprise-only/openshift/hacep-server-proc.adoc (hunk 2 above)
Comment (mramendi) [reply doc-content/enterprise-only/openshift/hacep-server-proc.adoc]: For the operation of the server is definitely the correct of these two

**Pull request merged**